LayoutTests/imported/w3c/web-platform-tests/css/css-contain/content-visibility/content-visibility-resize-observer-no-error.html

```diff
@@ -27,4 +29,6 @@
 
 <script>
 new ResizeObserver(() => {});
+requestAnimationFrame(takeScreenshot);
```

Comment:
This doesn't seem right, can we get down to why we're failing here? (potentially by comparing to what other browsers do)

I suspect it's either the fallback when the initial C-V callback hasn't happened yet, but I might be wrong.

Reply:
Yeah, I am experimenting a bit, there seem to be timing issues for WK1 test runner compared to WK2, some WK1 c-v tests seem to stop early compared to WK2 test run. When running mini-browser the end result of WK1 seems fine. So this part you can ignore for now (hopefully we can avoid making test changes just for WK1).

Reply:
Oh if this is only for WK1, I would just skip the tests there, we don't need to support new features for WK1. 

There are known timing issues for rendering update steps in WK1

Reply:
We have rdar://75508813 filed internally, no public bugzilla atm though

Reply:
I filed https://bugs.webkit.org/show_bug.cgi?id=259075

Reply:
I implemented https://drafts.csswg.org/css-contain/#cv-notes Step 4 so the WK1 test runner problem is gone now.
However next time I get WK1 problems I will remember the new bug, thanks!